src/Smalot/PdfParser/Element/ElementHexa.php

```diff
@@ -74,7 +74,7 @@ public static function decode($value, Document $document = null)
         $text   = '';
         $length = strlen($value);
 
-        if (substr($value, 0, 2) == '00') {
+        if (substr($value, 0, 2) === '00') {
```

Comment:
My php storm suggests replacing this line by : 
`if (strpos($value, '00') === 0) {`

Reply:
Indeed